fio/func.sh

```
@@ -1,30 +1,39 @@
 #!/bin/bash
 
-parse_iops() {
-        local OUTPUT_IOPS=$1
-        RAND_READ_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[0].read.iops_mean'| cut -f1 -d.`
-        RAND_WRITE_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[1].write.iops_mean'| cut -f1 -d.`
-        SEQ_READ_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[2].read.iops_mean'| cut -f1 -d.`
-        SEQ_WRITE_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[3].write.iops_mean'| cut -f1 -d.`
-        CPU_IDLE_PCT_IOPS=`cat $OUTPUT_IOPS | jq '.cpu_idleness.system' | cut -f1 -d.`
+parse_read_iops() {
+	local OUTPUT_IOPS=$1
+	RAND_READ_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[0].read.iops_mean'| cut -f1 -d.`
```

Shellcheck

[warning] RAND_READ_IOPS appears unused. Verify use (or export if used externally).

(SC2034)